manim/mobject/text/tex_mobject.py

```diff
@@ -447,8 +477,12 @@ class Tex(MathTex):
     """
 
     def __init__(
-        self, *tex_strings, arg_separator="", tex_environment="center", **kwargs
-    ):
+        self,
+        *tex_strings: Any,
```

Comment:
```suggestion
        *tex_strings: str,
```
?

Reply:
Right. At some point, I used `Any` for all the `*tex_strings` parameters, because of the strange `str(t)` in `MathTex._break_up_tex_strings()` which seemed to imply that anything could go in there. I reverted that change in `MathTex`, but forgot to do the same in `Tex`. Thanks for noticing!